index.js

```diff
@@ -1,2 +1,14 @@
+let num1 = 2;
+let num2 = 31;
+let multiply = 62;
```

Comment:
While I'm sure this does pass the test, can you think of another way to get 62 here using multiplication?